docs/decisions/0004-course-cache-use.rst

```diff
@@ -0,0 +1,37 @@
+4. Use of the LMS course caches
+###############################
+
+Status
+******
+
+**Accepted** *2024-02-26*
+
+Context
+*******
+Each course run ID in edx-platform is associated with another course ID. While for many courses, the mapping between
+course run ID and course ID is straight forward, i.e. edX+testX+2019 -> edX+testX, this is not the case for every
+course on edX. The discovery service is the source of truth for mappings between course run ID and course IDs, and
+string manipulation cannot be relied on as an accurate way to map between the two forms of ID.
+
+The learning-assistant CourseChatView NEED LINK accepts a course run ID as a query parameter, but parts of the python API defined
+in the learning assistant backend also require the course ID associated with a given course.
+
+Decision
+********
+In order to determine the mapping between a course run ID and course ID in the learning-assistant app, we will make
```

Comment:
I think we should also mention the course cache to get the skill names and title.